src/test/java/teammates/test/cases/action/InstructorFeedbackAddActionTest.java

```diff
@@ -153,17 +195,20 @@ public void testExecuteAndPostProcess() {
                 "TEAMMATESLOG|||instructorFeedbackAdd|||instructorFeedbackAdd|||true|||"
                 + "Instructor|||Instructor 1 of Course 1|||idOfInstructor1OfCourse1|||"
                 + "instr1@course1.tmt|||New Feedback Session "
-                + "<span class=\"bold\">(Course with minute offset timezone)</span> for Course "
+                + "<span class=\"bold\">(Course with DST time zone)</span> for Course "
                 + "<span class=\"bold\">[idOfTypicalCourse1]</span> created.<br>"
-                + "<span class=\"bold\">From:</span> 2012-01-31T18:30:00Z"
-                + "<span class=\"bold\"> to</span> 2014-12-31T18:30:00Z<br>"
-                + "<span class=\"bold\">Session visible from:</span> 1970-11-27T00:00:00Z<br>"
-                + "<span class=\"bold\">Results visible from:</span> 1970-01-01T00:00:00Z<br><br>"
+                + "<span class=\"bold\">From:</span> 2012-01-31T22:00:00Z"
+                + "<span class=\"bold\"> to</span> 2014-03-28T00:00:00Z<br>"
+                + "<span class=\"bold\">Session visible from:</span> 2011-12-31T22:00:00Z<br>"
+                + "<span class=\"bold\">Results visible from:</span> 1970-06-22T00:00:00Z<br><br>"
```

Comment:
@whipermr5 I was resolving the conflicts in #8447, Wanted to know how you chose these dates. Specially `session Visible from` and `results visible from` dates. I mean does these particular dates signify something?